design-documents/graph-ql/coverage/add-items-to-cart/AddConfigurableProductToCart.graphqls

```diff
@@ -0,0 +1,37 @@
+type Mutation {
+    addConfigurableProductsToCart(input: AddConfigurableProductsToCartInput): AddConfigurableProductsToCartOutput
+}
+
+input AddConfigurableProductsToCartInput {
+    cartId: String!
+    cartItems: [ConfigurableProductCartItemInput!]!
+}
+
+input ConfigurableProductCartItemInput {
+    sku: String!
+    quantity: Float!
+    configurable_options:[ConfigurableOptionInput!]!
+    customizable_options:[CustomizableOptionInput!]
+}
```

Comment:
Can't configurable_options be replaced by a single `simple_sku: Sku!` field?

Reply:
If we go with `simple_sku`, then the client on its end will have to map selected options combination to the SKU:
- Get a map during product request (option1, option2, optionN => simple SKU, ...)
- Calculate current simple SKU based on selected options
- Send simple SKU along with `add product to cart` request

Looks like more work for client app developers. I am probably missing something, please clarify why `simple_sku` is preferable.

Thanks for the review!

Reply:
I think there already is some form of simple_sku mapping in the frontend, especially if a frontend developer wishes to fetch images from the simple product etc. Or are there other strategies in that regard?

Reply:
@paales Can you help by providing the motivation for using `simple_sku` in lieu of sending along options?

AFAICT, it seems like, in any situation you'd be adding an item to a cart, you'd always have a list of the options (since the shopper needs to select them). So it feels like needing to track a `simple_sku` on the client-side might be work that is unnecessary. Granted, it's very little extra work, but I'm curious what the motivation would be for that > options list.

Reply:
In the past I've always found it much easier to supply a sku rather than a list of options that map to a simple product sku. I agree with @paales that usually the sku is already known on the frontend.